regression/cbmc/r_w_ok_inconsistent_dead/test-global-no-cp.desc

```diff
@@ -0,0 +1,10 @@
+FUTURE
+main.c
+--function test_global_pointer --pointer-check --no-simplify --no-propagation
+^EXIT=0$
+^SIGNAL=0$
+--
+^warning: ignoring
+--
+Tests that an inconsistent assumption involving a pointer to an out of scope
```

Comment:
:green_heart: Thanks

Reply:
Though an explanation of why FUTURE (and link to relevant ticket) would be good

Reply:
I think the test and the test description already provide this explanation. The FUTURE just indicates that the test does not currently pass.